### DISCUSSION TITLE
Which currency did 🇩🇪 use before the 💶?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#943] [Galaxy Trivia] [answered]
**Bonus:** What's on the front of the highest bill? What's on the back of the smallest one?

**Comment by tomerf** [comment]
Before the euro (€), Germany used the Deutsche Mark (DM) as its currency. On the front side of the DM 1,000 bill, there was a portrait of German politician and statesman, Friedrich von Schiller. On the back of the DM 5 bill, there was an image of a Germanic oak wreath, which symbolized strength and endurance.

**Comment by peterbaccaro** [comment, marked as answer]
Before the adoption of the euro, Germany used the **Deutsche Mark** (DM) as its currency. The Deutsche Mark was the official currency of West Germany starting in 1948 and became the currency of a reunified Germany in 1990 after the fall of the Berlin Wall. It remained in circulation until 2002 when Germany, along with several other European countries, replaced their national currencies with the euro as part of the European Union's monetary union.

The highest denomination banknote of the Deutsche Mark (DM) was the 1000 Deutsche Mark bill (DM 1000). On the front side of the DM 1000 bill, there is a **portrait of the German statesman and philosopher, Friedrich Wilhelm von Humboldt**. Humboldt was a significant figure in German intellectual and academic circles, known for his contributions to education and linguistics. The design also includes various ornate patterns and security features typical of banknotes.

The smallest denomination banknote of the Deutsche Mark (DM) was the 5 Deutsche Mark bill (DM 5). On the back side of the DM 5 bill, there is a **depiction of the Brandenburg Gate**. The Brandenburg Gate is a famous landmark in Berlin, Germany, and is one of the most well-known symbols of the city. The design of the banknote features an illustration of the Brandenburg Gate with the Quadriga sculpture on top, surrounded by ornate patterns and security features.
